pxtsim/libgeneric.ts

```diff
@@ -20,14 +20,17 @@ namespace pxsim {
         }
 
         toDebugString(): string {
+            return "[" + this.data.join(", ") + "]";
```

Comment:
If they have a really big array, this is going to take a lot of memory. I think you should stick with the bail out strategy

Reply:
Yes, makes sense. I rolled this back and just increased the maximum amount of characters we send to 100, which should be enough for most user-created arrays while making sure we don't throttle everything else.